source/libraries/NetworkConfigurations.ts

```diff
@@ -0,0 +1,46 @@
+import BN = require('bn.js');
+
+export type NetworkConfiguration = {
+    host: string;
+    port: number;
+    privateKey: string | undefined;
+    gasPrice: BN;
+}
+
+export type NetworkConfigurations = {
+    [networkName: string]: NetworkConfiguration;
+}
+
+export const networkConfigurations: NetworkConfigurations = {
+    ropsten: {
+        host: "ropsten.augur.net",
+        port: 8545,
+        privateKey: process.env.ROPSTEN_PRIVATE_KEY,
+        gasPrice: new BN(20)
+    },
+    kovan: {
+        host: "kovan.augur.net",
```

Comment:
Are we waiting on something to move Kovan and Ropsten over to the new subdomains?  I think they may already exist in CF but probably point at HAP which doesn't know about these two...

Reply:
I just needed to check with @epheph if they are good to switch.